sources/packages/web/src/composables/useFileUtils.ts

```diff
@@ -0,0 +1,28 @@
+import { StudentService } from "@/services/StudentService";
+import { StudentUploadFileDTO } from "@/types";
+
+/**
+ * File helper methods.
+ */
+export function useFileUtils() {
+  /**
+   * Used to download the document or file uploaded, which has the StudentUploadFileDTO structure
+   * @param studentDocument
+   */
+  const downloadDocument = async (studentDocument: StudentUploadFileDTO) => {
```

Comment:
Please create an interface specific to be used here with the specific properties needed: fileName, uniqueFileName.